### PR TITLE
DATAREDIS-674 - JedisClusterConnection to support zAdd with Set of Tuples.

### PR DESCRIPTION
**Pull request by clemong888**
Zadd more multiple elements in cluster mode was added. 
I have not added any unit tests because i can't seem to find where the test cases of JedisClusterZSetCommands.java are being written or if there are any at all at the moment.

- [◯] You have read the [Spring Data contribution guidelines](https://github.com/spring-projects/spring-data-build/blob/master/CONTRIBUTING.adoc).
- [◯ ] There is a ticket in the bug tracker for the project in our [JIRA](https://jira.spring.io/browse/DATAREDIS).
- [◯ ] You use the code formatters provided [here](https://github.com/spring-projects/spring-data-build/tree/master/etc/ide) and have them applied to your changes. Don’t submit any formatting related changes.
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [◯ ] You added yourself as author in the headers of the classes you touched. Amend the date range in the Apache license header if needed. For new types, add the license header (copy from another file and set the current year only).

### CODE REVIEW

File: src/main/java/org/springframework/data/redis/connection/RedisZSetCommands.java [hunk 1]
```diff
@@ -740,4 +745,34 @@ default Long zRemRangeByScore(byte[] key, double min, double max) {
 	 */
 	Set<byte[]> zRangeByLex(byte[] key, Range range, Limit limit);
 
+	/**
+	 * Convert tuples to map of bytes and double.
+	 * Bytes represents the value of the element and double is for the score.
+	 * @param tuples
+	 * @return
+	 */
+	default Map<byte[], Double> zAddArgs(Set<Tuple> tuples) {
```
Comment (mp911de) [inline src/main/java/org/springframework/data/redis/connection/RedisZSetCommands.java]: This interface is driver-agnostic and the change would affect users using the Lettuce driver. Rather move the tuple conversion into `JedisConverters`.

Comment (clemong888) [reply src/main/java/org/springframework/data/redis/connection/RedisZSetCommands.java]: Thanks for the comment. Yes, it's better to be in JedisConverters. I've made the changes. Please check

**Comment by mp911de** [comment]
Thanks. I'll take the PR from here.

**Comment by mp911de** [comment]
That's merged, polished and back-ported.

**Comment by clemong888** [comment]
Thank you!

**Pull request closed without merging**